contracts/PositionManager.sol

```diff
@@ -100,11 +96,24 @@ contract PositionManager is IPositionManager, ERC721Holder {
         }
     }
 
-    ///@notice remove awareness of _id UniswapV3 NFT
-    ///@param _id ID of the NFT to remove
-    function removePositionId(uint256 _id) public override {
+    ///@notice remove awareness of tokenId UniswapV3 NFT
+    ///@param tokenId ID of the NFT to remove
+    function removePositionId(uint256 tokenId) public override {
+        StorageStruct storage Storage = PositionManagerStorage.getStorage();
+        try
```

Comment:
Can you explain/check this try-catch block?

Reply:
We need to check if the position is really not in the position manager anymore, but if the NFT has been burn, nonfungiblePositionManager.ownerOf() reverts. To avoid this we do a try-catch:
- if the function returns, we check that owner is not positionManager, so we can remove the ID from the list
- if the function reverts, the NFT has been burn (or does not exist), so we can remove as well the ID